server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/component/Component.java

```diff
@@ -79,6 +79,14 @@ public boolean isViewsType() {
   String getKey();
 
   /**
+   * Returns the key as it will be displayed in the ui.
+   * If legacy branch feature is used, the key will contain the branch name
+   * If new branch feature is used, the key will not contain the branch name
+   */
+  // TODO to be renamed getKey() and rename existing getKey to getDbKey
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)